yeatmanlab/create-roar-app
# Add a section on common issues

### Notes

* Create a new section on common issues and update the navigation page.
* Move the section on advice for asset files and audio stimuli to common issues.
* Small changes such as content tabs and typo.

### Testing

* Install [MkDocs](https://www.mkdocs.org/) by following the [documentation](https://www.mkdocs.org/user-guide/installation/).
* Use `mkdocs serve` and manually check the changes.
